applications/hangups/src/hangups_channel_destroy.erl

```diff
@@ -32,28 +32,58 @@
 %%
 %% @end
 %%--------------------------------------------------------------------
--spec handle_req(wh_json:object(), wh_proplist()) -> any().
-handle_req(JObj, _Props) ->    
+-spec handle_req(wh_json:object(), wh_proplist()) -> 'ok'.
+handle_req(JObj, _Props) ->
     'true' = wapi_call:event_v(JObj),
     HangupCause = wh_json:get_value(<<"Hangup-Cause">>, JObj, <<"unknown">>),
     case lists:member(HangupCause, ?IGNORE) of
         'true' -> 'ok';
         'false' ->
-            AccountId = wh_json:get_value([<<"Custom-Channel-Vars">>, <<"Account-ID">>], JObj),
-            lager:debug("abnormal call termination: ~s", [HangupCause]),
-            wh_notify:system_alert("~s ~s to ~s (~s) on ~s(~s)"
-                                   ,[wh_util:to_lower_binary(HangupCause)
-                                     ,find_source(JObj)
-                                     ,find_destination(JObj)
-                                     ,find_direction(JObj)
-                                     ,find_realm(JObj, AccountId)
-                                     ,AccountId
-                                    ]
-                                   ,maybe_add_hangup_specific(HangupCause, JObj)
-                                  ),
-            add_to_meters(AccountId, HangupCause)
+            alert_about_hangup(HangupCause, JObj)
     end.
 
+
+-spec alert_about_hangup(wh_json:object(), ne_binary()) -> 'ok'.
+alert_about_hangup(HangupCause, JObj) ->
+    lager:debug("abnormal call termination: ~s", [HangupCause]),
+    EmailTrue = whapps_config:get_is_true(?APP_NAME, <<"enable_email_alerts">>, 'true'),
+    SUBUrl    = whapps_config:get_string(?APP_NAME, <<"subscriber_url">>, <<>>),
```

Comment:
Use `get_string/2` here, since the type cast only applies when a value is found. So your call here will return either the list value of "subscriber_url" or an empty binary. In the case clause below, you test for `<<>>` but in `alert_about_hangup__POST/3` you test for `""` which won't match `<<>>`.

With `get_string/2`, you'd get a list or `'undefined'` which is more explicit to test for.